DependencyInjection/HttplugExtension.php

```diff
@@ -56,6 +61,10 @@ protected function configureClients(ContainerBuilder $container, array $config)
                 $first = $name;
             }
 
+            if (isset($config['_inject_journal_plugin'])) {
+                array_unshift($arguments['plugins'], 'httplug.plugin.history');
```

Comment:
what if somebody was already adding the history plugin? should we array_search ('httpplug.plugin.history') to avoid that?